spec/unit/artifactserver_spec.rb

```diff
@@ -56,6 +56,59 @@ module CookbookOmnifetch
       }
       expect(public_repo_location.lock_data).to eq(expected_data)
     end
+
+    context "when asked to install a new cookbook" do
+
+      let(:http_client) { double("Http Client") }
+
+      let(:test_root) { Dir.mktmpdir(nil) }
+
+      let(:storage_path) { File.join(test_root, "storage") }
+
+      let(:cache_path) { File.join(test_root, "cache") }
+
+      let(:path) { fixtures_path.join('cookbooks') }
```

Comment:
this could be more descriptive, like `cookbook_fixtures_path`. I had to refer back to this to figure out what the test was doing.